### DISCUSSION TITLE
Shows discharging instead of charging?

### DISCUSSION THREAD

**Discussion by BatteryTester3** [Louisvdw/dbus-serialbattery#745] [Q&A] [answered]
Having issues with Daly bms on the venus os showing discharging instead of charging. How do I fix this? I connected it correctly, B to battery and P to load.

**Comment by drtinaz** [comment, marked as answer]
This is a common issue with Daly. Look at the install notes. There are instructions there how to add the setting to config.ini to correct this.

**Reply by BatteryTester3** [reply]
thanks!!
